specs/lib/stalePullRequestSpec.js

```diff
@@ -8,6 +8,18 @@ var RepositorySettings = require('../../lib/RepositorySettings');
 
 describe('stalePullRequest', function () {
     var repositories;
+    var commitsData;
+
+    beforeAll(function () {
```

Comment:
Is there a reason why `commitsData` is initialized in `beforeAll` while `repositories` is initailized in `beforeEach`? Otherwise they should probably be initialized consistently.